app/pipes/render-template.ts

```diff
@@ -27,11 +27,18 @@ env.addFilter('split', function(x: string) {
   }
 });
 
+env.addFilter('search_link', function(urlEncodedParams: string, theme_id: string) {
+  return '//next.obudget.org/s/?' + urlEncodedParams + (theme_id ? '&theme=' + theme_id : '');
+});
 
+env.addFilter('item_link', function(doc_type, entity_kind, entity_id, theme_id: string) {
```

Comment:
It's better to use the document id here and not take the entity_kind, entity_id parameters which are only relevant for entities.